3515/2/helper_modules/user.rb

```diff
@@ -0,0 +1,7 @@
+require 'ohm'
+class User < Ohm::Model
```

Comment:
Add an empty line before the class definition in case of any `require`s for all the classes on this PR